### DISCUSSION TITLE
Question about "NormalsSolverVCE"

### DISCUSSION THREAD

**Discussion by Namesume** [groops-devs/groops#197] [Q&A]
Dear Developers,

I'm trying to recover the gravity-field model by using groops. I've successfully build the normal equations and now trying to solve them by using "NormalsSolverVCE".  But I could not output the accelerometer bias in solution though I set the "spherical harmonics", "accelerometer bias" and "accelerometer scale". I've already checked the all the output of this program, tried to set different options of this program, but i could not output the accbias(the parameterName only contains spherical harmonics and accscale).
Could you please tell me how can I get the accbias of the normal equations? Maybe I haven't set the program correctly yet.

Like always thank you in advance for your time and help.

With kind regards,
Shek

**Comment by tmayerguerr** [comment]
Hi Shek,
Unfortunately, I cannot see from your question where the problem might lie. Can you just post your config file?
regards
Torsten

**Reply by Namesume** [reply]
Dear Torsten,
Thank you so much for your reply. I would like to describe my problem again and attach the pictures. Now, I'm trying to recover a daily gravity-field model (20 degree) by using the orbit of GRACEFO.The first picture shows the Global variables of bias and scale. Whether the scales of three direction were estimated or not, the biases were always estimated. the second picture shows the setting of 'VCE' program. Then the third and fourth pictures show the normal parameter names of two different results. The third picture shows the result of not estimating the scale, while the fourth picture shows the result of estimating the scale. I could not get the bias though I estimated the three biases. The program was made according to 'scenarioGravityFromKinematicOrbits', and i truned 'NormalsEliminate' off.
Like always thank you in advance for your time and help.
With kind regards,
Shek

Picture1
![acc bias and acc scale](https://github.com/groops-devs/groops/assets/114293746/6882aefb-7889-4667-bbaa-833775a2b00b)

Picture2
![VCE](https://github.com/groops-devs/groops/assets/114293746/3fbc5cdf-cf70-45eb-8fff-2de4db1c17a8)

Picture3
![Don't esitimated acc scale](https://github.com/groops-devs/groops/assets/114293746/e736d725-a978-416e-a486-f475a544bd84)

Picture4
![Esitimated acc scale](https://github.com/groops-devs/groops/assets/114293746/7ec72c08-16db-4912-94e4-0a32730abb02)

**Comment by Namesume** [comment]
自动回复：您好，我已收到您的邮件。

**Comment by tmayerguerr** [comment]
Hi Shek,

The problem is that the bias parameters are defined arc-wise (perArc). Arc parameters are always eliminated in advance (see [normalEquation](https://groops-devs.github.io/groops/html/normalEquationType.html#design)). Unfortunately, there is no option to change this behaviour. The only possibility I see at the moment is to change the bias parameterisation (e.g. linear splines every 20 minutes).

regards
Torsten

**Reply by Namesume** [reply]
Dear Torsten,

Thank you so much for your advice. I will check the note about normalEquation. I have two more question about the accelerometer bias and scale. There exists a program call 'InstrumentAccelerometerEstimateBiasScale', and I carefully reviewed the introduction of it.  Knowing that it aims to calibrate the bias and scale with respect to simulated accelerometer, which is different from the Recovery theory (the bias and scale are calculated in solving normal equation, synchronously with spherical harmonics). Is this program used to simply correct system deviations? Can I use this program to get the bias and scale?

Like always thank you in advance for your time and help.
With kind regards,
Shek
